controller/comments.go

```diff
@@ -112,10 +113,16 @@ func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		if *identityID != cm.CreatedBy {
-			// need to use the goa.NewErrorClass() func as there is no native support for 403 in goa
-			// and it is not planned to be supported yet: https://github.com/goadesign/goa/pull/1030
-			return jsonapi.JSONErrorResponse(ctx, goa.NewErrorClass("forbidden", 403)("User is not the comment author"))
+		wi, err := appl.WorkItems().LoadByID(ctx.Context, cm.ParentID)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+		authorized, err := authz.Authorize(ctx, wi.SpaceID.String())
```

Comment:
thanks, @alexeykazakov, now separated into 2 transactions in https://github.com/almighty/almighty-core/pull/1377/commits/6e1ea960781470fc90508cc612c428092ce2f5f8